letsencrypt/cli.py

```diff
@@ -1040,8 +1040,7 @@ def main(cli_args=sys.argv[1:]):
 
     # note: arg parser internally handles --help (and exits afterwards)
     plugins = plugins_disco.PluginsRegistry.find_all()
-    parser, tweaked_cli_args = create_parser(plugins, cli_args)
-    args = parser.parse_args(tweaked_cli_args)
```

Comment:
I was worried that there might have been a subtle bug of that sort (some call needing access to the command line args before they were reordered and a default verb introduced), but I think I was mistaken.  Tweaked_cli_args was introduced in 44d7ac2dce2691116da0ba4fa88d37d7fde89134 because of the opposite problem -- the Namespace being made from the unedited arguments.